package/score-worker/src/main/java/io/cloudslang/schema/ConfValue.java

```diff
@@ -50,6 +50,10 @@ public void register(Element element, ParserContext parserContext){
 	}
 
 	private Object fromString(String value) {
+		String sysValue = System.getProperty("cloudslang.worker." + name);
+		if(sysValue != null) {
+			value = sysValue;
+		}
```

Comment:
to complex...
it's better to leave it with : 
value = System.getProperty("cloudslang.worker." + name,value);